operator/config/manager/kustomization.yaml

```diff
@@ -3,6 +3,5 @@ kind: Kustomization
 images:
 - name: controller
   newName: pulumi/pulumi-kubernetes-operator-v2
-  newTag: latest
```

Comment:
Is this intentional?

Reply:
I don't think so. Have reverted this change.